controllers/tradeHistory.js

```diff
@@ -0,0 +1,22 @@
+const postHistory = async (req, res) => {
+	const trade_id = req.body.trade_id;
+
+	const tradeResult = await axios.post(
+		"https://sandbox.tappaysdk.com/tpc/transaction/trade-history",
+		{
+			partner_key:
+				"partner_wcxH5GX2HMLk9p2WpfOYSWseWOyn0mE0K1VwqjIZAgDGCsRZ4BEqMCaL",
+			rec_trade_id: trade_id,x
+		},
+		{
+			headers: {
+				"Content-Type": "application/json",
+				"x-api-key":
+					"partner_wcxH5GX2HMLk9p2WpfOYSWseWOyn0mE0K1VwqjIZAgDGCsRZ4BEqMCaL",
+			},
+		}
+	);
+
+	return tradeResult;
+};
+export {postHistory}
```

Comment:
Btw if you're curious why github warns you about having no line in the end of file you can read this https://stackoverflow.com/questions/5813311/whats-the-significance-of-the-no-newline-at-end-of-file-log

<img width="912" alt="Screen Shot 2022-05-20 at 10 08 39 PM" src="https://user-images.githubusercontent.com/49668442/169545883-67bd6326-07e9-4916-806d-eb9087d84e86.png">

But to me it doesn't matter whether to add or not the line, so it's up to you